src/View/mainpage/DrawROIWindow/UIDrawROIWindow.py

```diff
@@ -51,14 +53,15 @@ def setup_ui(self, draw_roi_window_instance,
         self.drawingROI = None
         self.slice_changed = False
         self.drawing_tool_radius = INITIAL_DRAWING_TOOL_RADIUS
-        self.keep_empty_pixel = False
+        self.keep_empty_pixel = True  # This constant will not change
```

Comment:
I think we need a guaranteed decision on whether this will stay in or not, otherwise, it'll end up getting lost in the code.

Reply:
PO, wanted to play with this feature before deciding if a "keep empty pixel" toggle is needed. This will be removed if they decide it is not needed.

Reply:
Sounds good, create a ticket to follow up later